### PR TITLE
Alternative languages

### PR DESCRIPTION
**Pull request by apfelbox**

### CODE REVIEW

File: src/Story/StoryMetaData.php [hunk 1]
```diff
@@ -154,8 +155,43 @@ private function parseDate (string $date) : \DateTimeImmutable
...
+	 * @return array<string, string> locale => full_slug
+	 */
+	public function getTranslatedDocumentsMapping () : array
+	{
+		$mapping = [];
+
+		foreach ($this->getAlternateLanguages() as $alternateLanguage)
+		{
+			$slug = $alternateLanguage["full_slug"];
+			$locale = \mb_substr($slug, 0, \strpos($slug, "/") ?: null);
```
Comment (keichinger) [inline src/Story/StoryMetaData.php]: Should we store the locale as its own field within the story?

I'd love to get rid of the additional extraction and evaluation in here and move it into the document generation, if easily possible.

**Review by keichinger — DISMISSED** [review]

File: src/Story/StoryMetaData.php (hunk 1 above)
Comment (apfelbox) [reply src/Story/StoryMetaData.php]: We don’t get the full story here but just some metadata. 

Or do you mean to move the parsing one level up into the other getter?

Comment (keichinger) [reply src/Story/StoryMetaData.php]: Yes, just the parsing

Comment (apfelbox) [reply src/Story/StoryMetaData.php]: Understood. Is updated 👍

**Review by keichinger — APPROVED** [review]

**Pull request merged**